packages/e2e-tests/config/setup-test-framework.js

```diff
@@ -155,6 +155,14 @@ function observeConsoleLogging() {
 			return;
 		}
 
+		// As of WordPress 5.3.2 in Chrome 79, navigating to the block editor
+		// (Posts > Add New) will display a console warning about
+		// non - unique IDs.
+		// See: https://core.trac.wordpress.org/ticket/23165
+		if ( text.includes( '[DOM] Found 2 elements with non-unique id #_wpnonce' ) ) {
```

Comment:
Given that the "2" here might fluctuate depending on how many duplicates there are, we might want to make this a little more tolerant.

Either:

```suggestion
		if ( /\[DOM\] Found \d+ elements with non-unique id #_wpnonce/.test( text ) ) {
```

Or:

```suggestion
		if ( text.includes( 'elements with non-unique id #_wpnonce' ) ) {
```

Open question about whether we want to be tolerant about `_wpnonce` ID. I think it's something we'd want to consider on a case-by-case basis, so probably fine to leave as strict consideration. (I guess this could potentially be used as an argument that evaluating strictly "2" elements could make sense as well 🤷‍♂ )

Reply:
Thanks for noting that 

Id assumed we would want to be very specific about the number in the error. Having 3 elements might point to a a different error and mask it.

On reflection however the wpnonce makes it fairly specific so allowing any count of duplicates seems sensible.

Ill amend.

Reply:
@aduth I've pushed a fix accepting your 2nd suggestion as I felt that was the simplest approach.

Just need a 👍 if you're happy to merge once tests are green.